templates/plugins/jscomments/hypercomments.html.twig

```diff
@@ -0,0 +1,30 @@
+{#
+    HyperComments - IntenseDebate is a feature-rich comment system for
+    WordPress, Blogger, Tumblr and many other blogging/CMS platforms.
+
+    @see https://www.hypercomments.com/
+ #}
+<div id="jscomments">
+    <div id="hypercomments_widget"></div>
+
+    <script type="text/javascript">
+        _hcwp = window._hcwp || [];
+        _hcwp.push({
+            widget: "Stream",
+            widget_id: '{{ widget_id|e('js') }}',
+            social: '{{ social|e('js') }}'
+        });
+        (function () {
+            if ("HC_LOAD_INIT" in window)return;
+            HC_LOAD_INIT = true;
+            var lang = (navigator.language || navigator.systemLanguage || navigator.userLanguage || "en").substr(0, 2).toLowerCase();
+            var hcc = document.createElement("script");
+            hcc.type = "text/javascript";
+            hcc.async = true;
+            hcc.src = ("https:" == document.location.protocol ? "https" : "http") + "://w.hypercomments.com/widget/hc/90285/" + lang + "/widget.js";
+            var s = document.getElementsByTagName("script")[0];
+            s.parentNode.insertBefore(hcc, s.nextSibling);
+        })();
+    </script>
+    <a href="http://hypercomments.com" class="hc-link" title="comments widget">comments powered by HyperComments</a>
```

Comment:
In principle, capitalize "comments powered by HyperComments". Furthermore, what do you think to add this as a translate key to `languages.yaml` and translate it for every language?